### ISSUE TITLE
Upgrading PHP 7.0 from 7.0.3 to 7.04 or 7.0.5 breaks ownCloud

### ISSUE THREAD

**Issue by RussianNeuroMancer** [owncloud/core#23746]
### Steps to reproduce
1. Setup ownCloud 9.0 on Ubuntu Server 16.04 x86_64 with Apache 2 (build with enabled HTTP/2 support) PHP-FPM 7.0.3 (opcache enabled) and Redis.
3. Check if ownCloud is working
2. Upgrade PHP from 7.0.3 to 7.0.4 (from system repository) or 7.0.5 (from ppa:ondrej/php)

### Expected behaviour
ownCloud works as before PHP upgrade

### Actual behaviour
For example phpinfo(); is works (so Apache<->PHP-FPM tandem s working) but access to any page related to ownCloud gives just empty output. I tried to enable php debug info in fpm's php.ini,  restart services, reboot, etc. but this doesn't change outcome.

### Server configuration
**Operating system**: Ubuntu Server 16.04

**Web server:** Apache 2 build with enabled HTTP/2 support from ppa:ondrej/apache2

**Database:** PostgreSQL 9.5.2

**PHP version:** 7.0.4 or 7.0.5

**ownCloud version:** 9.0.0

**Updated from an older ownCloud or fresh install:** from 8.2.2

**Where did you install ownCloud from:** owncloud repository for Ubuntu 15.10 (since owncloud-deps-php7 package here contain incorrect dependencies and can not be installed on 15.10 or 16.04, required PHP 7 packages (php php-cli php-common php-curl php-fpm php-gd php-igbinary php-json php-mbstring php-pgsql php-redis) was installed manually).

**Signing status (ownCloud 9.0 and above):**

```
No errors have been found.
```

**List of activated apps:**

```
  - activity: 2.2.1
  - calendar: 1.0
  - comments: 0.2
  - contacts: 1.0.0.0
  - dav: 0.1.5
  - documents: 0.12.0
  - federatedfilesharing: 0.1.0
  - federation: 0.0.4
  - files: 1.4.4
  - files_pdfviewer: 0.8
  - files_sharing: 0.9.1
  - files_texteditor: 2.1
  - files_trashbin: 0.8.0
  - files_versions: 1.2.0
  - firstrunwizard: 1.1
  - gallery: 14.5.0
  - notifications: 0.2.3
  - provisioning_api: 0.4.1
  - qownnotesapi: 0.4.3
  - systemtags: 0.2
  - templateeditor: 0.1
  - updatenotification: 0.1.0
```

**The content of config/config.php:**

```
{
    "system": {
        "instanceid": "ocrmtmlqgknc",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "***REMOVED SENSITIVE VALUE***",
            "***REMOVED SENSITIVE VALUE***"
        ],
        "datadirectory": "\/srv\/data",
        "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "pgsql",
        "version": "9.0.0.19",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "filelocking.enabled": "true",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "\/var\/run\/redis\/redis.sock",
            "port": 0,
            "timeout": 0
        },
        "asset-pipeline.enabled": true,
        "enabledPreviewProviders": {
            "0": "OC\\Preview\\PNG",
            "1": "OC\\Preview\\JPEG",
            "2": "OC\\Preview\\GIF",
            "3": "OC\\Preview\\BMP",
            "4": "OC\\Preview\\XBitmap",
            "5": "OC\\Preview\\MP3",
            "7": "OC\\Preview\\MarkDown",
            "8": "OC\\Preview\\MSOffice2003",
            "9": "OC\\Preview\\MSOffice2007",
            "10": "OC\\Preview\\MSOfficeDoc",
            "11": "OC\\Preview\\OpenDocumen",
            "12": "OC\\Preview\\PDF"
        },
        "maintenance": false,
        "loglevel": 2,
        "updatechecker": false,
        "theme": "",
        "mail_smtpmode": "smtp",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "ssl",
        "mail_smtpauthtype": "PLAIN",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***"
    }
}
```

**Are you using external storage, if yes which one:** local

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** Firefox 45.01, 46, 47.

**Operating system:** Kubuntu 16.04 x86_64

#### Web server error log
```
[Sat Apr 01 21:47:17.976717 2016] [:notice] [pid 284:tid 139673393604480] FastCGI: process manager initialized (pid 284)
[Sat Apr 01 21:47:17.983656 2016] [mpm_worker:notice] [pid 281:tid 139673393604480] AH00292: Apache/2.4.18 (Ubuntu) mod_fastcgi/mod_fastcgi-SNAP-0910052141 OpenSSL/1.0.2g configured -- resuming normal operations
[Sat Apr 01 21:47:17.986596 2016] [core:notice] [pid 281:tid 139673393604480] AH00094: Command line: '/usr/sbin/apache2'
```

#### ownCloud log (data/owncloud.log)
```
emtpy
```

#### Browser log
```
Server return empty page, so there is nothing
```

**Comment by tflidd** [comment]
Did you enable logging for php?
https://forum.owncloud.org/viewtopic.php?f=17&t=7330

**Comment by RussianNeuroMancer** [comment]
I tried (changed php_flag[display_errors], php_admin_value[error_log], php_admin_flag[log_errors] options in /etc/php/7.0/fpm/pool.d/www.conf) but I not sure if I done everything right for PHP-FPM config (because logs remain empty) so since issue is also reproducible with occ command line tool (empty output too) this time I configure php-cli instead. Please check configuration:

```
~$ cat /etc/php/7.0/cli/php.ini | grep '^[^#;]' | grep 'display_errors\|log_errors\|error_log\|error_reporting'
error_reporting = E_ALL
display_errors = On
log_errors = On
log_errors_max_len = 1024
error_log = /var/log/php-errors.log
~$ sudo -u www-data php justphpinfofile.php | grep 'display_errors\|log_errors\|error_log\|error_reporting'
display_errors => STDOUT => STDOUT
error_log => /var/log/php-errors.log => /var/log/php-errors.log
error_reporting => 32767 => 32767
log_errors => On => On
log_errors_max_len => 1024 => 1024
opcache.error_log => no value => no value
```
opcache is not enabled in cli config, /var/log/php-errors.log is created, owned by www-data:www-data, but still empty after running "sudo -u www-data php occ" (which also gives empty output) while php itself is working (parsing justphpinfofile.php successfully in cli and fpm modes).

**Comment by RussianNeuroMancer** [comment]
Issue is reproducible with ownCloud 9.0.1 too.

**Comment by tflidd** [comment]
If you run the occ-command from your terminal it should give you either an output of the menu or an error. Running the cli-version, we know that is neither an issue of php-fpm nor http/2. Is other code slightly more complex than phpinfo() running (phpmysql, wordpress, ...)?

**Comment by RussianNeuroMancer** [comment]
Ok, I tested WordPress with this combination:
PHP 7.0.4 from Ubuntu repository
ownCloud 9.0.1 (deb package from official repository)
WordPress 4.4.2 from Ubuntu repository
ownCloud and WordPress running with same Apache and PHP, in same container with same configs, etc.

Results:
ownCloud - same result as in comment above, just empty page.
WordPress - so far installation, admin panel, main page and posts publishing works fine, I doesn't notice anything unusual. If additional test required please let me know.

**Comment by azlux** [comment]
Same here after php7 update.
If you check the changelog of php : 
PHP 7 ChangeLog
Version 7.0.5
31 Mar 2016
    Core:
        Huge pages disabled by default.

Maybe it's that ?

**Comment by azlux** [comment]
Ok check you data/owncloud.log
The new php have divide somes librairies like php7.0-zip and php7.0-xml from the php7-commun package. Install them !

**Comment by RussianNeuroMancer** [comment]
Yep, that helps. So as I understand this packages should be added to owncloud-deps-php7 dependencies?

**Comment by DeepDiver1975** [comment]
@jnweiger does this affect our packaging? THX

**Comment by azlux** [comment]
and you can add the package php7.0-mbstring too. It's needed for contacts and now separated from php7.0-commun

**Comment by PVince81** [comment]
Is this still an issue with 9.0.4 ? There were several fixes in recent OC 9.0 to make it work better with later PHP 7 versions.
